src/testdir/test_diffmode.vim

```diff
@@ -1035,6 +1035,13 @@ func Test_diff_screen()
   call WriteDiffFiles(buf, [], [0])
   call VerifyBoth(buf, "Test_diff_22", "")
 
+  call WriteDiffFiles(buf, ['?a', '?b', '?c'], ['!b'])
+  call term_sendkeys(buf, ":set diffopt+=linematch:30\<cr>")
+  " write the contents to a screen dump?
+  call term_dumpwrite(buf, "Test_diff_23")
```

Comment:
You should remove the calls to the term_sendkeys() and term_dumpwrite() functions above.  Uncomment the below call to VerifyInternal() and run the test.  The test will fail and the screen dump will be in the testdir/failed directory.  You can verify the screen dump by loading it using the term_dumpload() function.  Once you have manually verified the screen dump, you can copy that file to the testdir/dumps directory.  Now the test should pass.